src/layer/__tests__/cache.ts

```diff
@@ -113,3 +113,188 @@ describe('Testing caching', () => {
     expect(mockNetwork.history.post.length).toBe(2);
   });
 });
+
+describe('Testing cache targets', () => {
+  beforeEach(() => {
+    Object.assign(global, makeServiceWorkerEnv(), fetch);
+    jest.resetModules();
```

Comment:
The meaning of these two lines is not apparent to me... If they are needed, could you maybe add a comment explining their purpose?

Reply:
I have removed `jest.resetModules` and added a comment to the remaining lines